js/teams.js

```diff
@@ -0,0 +1,41 @@
+$(function(){
```

Comment:
vies na to napisat nejaku funkciu? toggleTeam(teamName) {.....} a dnu vo funkcii spravis len hide show a pouzijes premennu teamName. skus na to pozriet, pripadne poobede na to mozeme pozret spolu. ale najprv skus sam poguglit.